hydrolysis-analyzer.html

```diff
@@ -65,6 +65,10 @@
         notify:   true,
       },
 
+      permissive: {
```

Comment:
fsResolver is never correct for this usecase. I'd rather not let someone specify it.